Add polish

jasmussen opened the pull request:
This PR contains 3 sets of fixes. 

The first polishes the link dialog a bit, by using color variables. Notably it fixes a regression with the Image link:

<img width="439" alt="screen shot 2018-04-12 at 11 01 05" src="https://user-images.githubusercontent.com/1204802/38670681-c2b82034-3e49-11e8-9214-136180edd158.png">

There's still an issue here, though, as soon as you start typing in the link dialog for images, it disappears. I suspect isEditing is kicking in and hiding the block toolbars. CC: @noisysocks 

The second part unifies the styles between readmore and pagination:

<img width="721" alt="screen shot 2018-04-12 at 11 38 52" src="https://user-images.githubusercontent.com/1204802/38670753-f81a7d30-3e49-11e8-8eef-bf243be0425b.png">

The third is a fix to make sure that placeholders fit in tight spots, by including a shorter Media Library label. It also makes the font sizes on those buttons consistent with all other Gutenberg buttons:

<img width="579" alt="screen shot 2018-04-12 at 12 03 17" src="https://user-images.githubusercontent.com/1204802/38670785-0e2569c8-3e4a-11e8-8a22-64073980bc8b.png">

Commit:
Polish link dialog.

This uses variables for colors and shadows, and also fixes a regression with the link dialog when linking an image.

However there's still a bug here, where as soon as you start typing in the image link dialog, the entire thing disappears and you can't type. This is possibly due to isEditing mode being invoked there. Can you take a look, @noisysocks @karmatosed?

Commit:
Unify markup between More block and Pagination block

This adds a white background, and some things, to the Pagination block. This will only be visible if a user loads a stylesheet into the editor.

Fixes https://github.com/WordPress/gutenberg/pull/1467#discussion_r180033504, cc @swissspidy.

Commit:
Unify button sizes, and slim down "Media Library".

This makes the placeholders fit better in small breakpoints.

mtias commented:
> The third is a fix to make sure that placeholders fit in tight spots, by including a shorter Media Library label.

Love this.

Commit:
Change left arrow on breadcrumbs.

This makes the left arrow consistent with the one used when creating links on images.

mtias commented:
Looks good to me. 🚢

jasmussen commented:
Woop woop!

noisysocks commented:
> I suspect isEditing is kicking in and hiding the block toolbars. CC: @noisysocks

It looks like this regression was introduced in https://github.com/WordPress/gutenberg/pull/5513. I'll create an issue.

aduth commented:
>>I suspect isEditing is kicking in and hiding the block toolbars. CC: @noisysocks
>
>It looks like this regression was introduced in #5513. I'll create an issue.

Related: #5942, #5894
